corehq/apps/app_manager/templates/app_manager/partials/xpathValidator.html

```diff
@@ -1,7 +1,9 @@
 {% load hq_shared_tags %}
 <script src="{% static 'xpath/lib/biginteger.js' %}"></script>
```

Comment:
biginteger isn't included in the xpath 2.0.0 library. it should be included from bower